src/flows/components/QueryTextPreview.tsx

```diff
@@ -1,24 +1,51 @@
-import React, {FC, useContext} from 'react'
+// Libraries
+import React, {FC, useContext, useRef} from 'react'
+
+// Components
 import {Form} from '@influxdata/clockface'
-import FluxEditor from 'src/shared/components/FluxMonacoEditor'
+import FluxEditorMonaco from 'src/shared/components/FluxMonacoEditor'
+
+// Utils
 import {formatQueryText} from 'src/flows/shared/utils'
+
+// Contexts
 import {PopupContext} from 'src/flows/context/popup'
 
+// Types
+import {EditorType} from 'src/types'
+
+// Constants
+import {MONACO_LINE_HEIGHT} from 'src/shared/constants/fluxEditor'
+const MAX_PREVIEW_HEIGHT = 300
+
 const QueryTextPreview: FC = () => {
+  const editor = useRef<EditorType>(null)
   const {data} = useContext(PopupContext)
 
-  const formattedQueryText = formatQueryText(data.query)
+  const handleSetEditorInstance = (monacoEditor: EditorType): void => {
+    editor.current = monacoEditor
+  }
+
+  const {text, lineCount} = formatQueryText(data.query)
```

Comment:
instead of all this noise, it'd be better to make the autogrow work correctly right?